docker/docker-compose.yml

```diff
@@ -38,9 +38,11 @@ services:
       - couchdb:/var/lib/couchdb
 
   rabbitmq:
-    image: rabbitmq:3.8.16
+    image: rabbitmq:3.8.16-management
     volumes:
       - rabbitmq:/var/lib/rabbitmq:z
+    ports:
+      - "15672:15672"
```

Comment:
Does this need to be defined in docker-server-configs?

Reply:
This file is for local development only not used in prod.

Reply:
Ah, duh. Yes.